_posts/0000-01-02-Code-panther.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"The way I see it, if you want the rainbow, you gotta put up with the rain." - Dolly Parton